components/KitButtonSupreme.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { View, StyleSheet, Image, TouchableOpacity } from "react-native";
+import Colors from "../constants/Colors";
+import KitText from "./KitText";
+
+// Button can either be 'filled' or 'outlined
+// Actual correct solution is to have a type checker + documentation but it be like that
+export default function KitButtonSupreme(props) {
+  let type = props.type || "filled";
+  if (props.type !== "filled" && props.type !== "outlined") {
```

Comment:
This seems like a repeated check with line 9. 

Consider using a `switch` statement instead? Something along the lines of
```
switch (props.type){
    case ("outlined"):
        type = "outlined";
        break;
    default:
        type = "filled";
}
```

Reply:
👍